augly/audio/functional.py

```diff
@@ -1269,3 +1269,44 @@ def to_mono(
     )
 
     return audutils.ret_and_save_audio(aug_audio, output_path, sample_rate)
+
+
+def loop_audio(
```

Comment:
two tiny nits:
(a) let's name this function `loop` for consistency with other libraries :)
(b) instead of adding this function to the end of `functional.py`, could you insert it in alphabetically with respect to the other function names in this file?